### PR TITLE
TAS-65/project-structure

### PR DESCRIPTION
**Pull request by Brendan-bx**
Creation of Project Structure with all dependencies and file structure

### CODE REVIEW

File: apps/admin/README.md
Comment (Limerio) [inline apps/admin/README.md:1]: Empty the file and remplace the content by this
```md
# Admin
```

**Review by WookiesRpeople2 — COMMENTED** [review]
For me this looks good you can go ahead and merge @Limerio

File: apps/admin/index.html
Comment (Limerio) [inline apps/admin/index.html:1]: Change the content title tag to `Administration - ALittleByte`

File: apps/admin/src/index.css
Comment (Limerio) [inline apps/admin/src/index.css:1]: Remove this file

File: apps/admin/src/assets/react.svg
Comment (Limerio) [inline apps/admin/src/assets/react.svg:1]: Remove this file

File: apps/admin/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useState } from "react"
+import reactLogo from "./assets/react.svg"
+import viteLogo from "/vite.svg"
+import "./App.css"
+
+function App() {
+	const [count, setCount] = useState(0)
```
Comment (Limerio) [inline apps/admin/src/App.tsx]: Remove this line

File: apps/admin/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { useState } from "react"
+import reactLogo from "./assets/react.svg"
+import viteLogo from "/vite.svg"
+import "./App.css"
```
Comment (Limerio) [inline apps/admin/src/App.tsx]: remove those lines

File: apps/admin/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+					count is {count}
+				</button>
+				<p>
+					Edit <code>src/App.tsx</code> and save to test HMR
+				</p>
+			</div>
+			<p className="read-the-docs">
+				Click on the Vite and React logos to learn more
+			</p>
+		</>
```
Comment (Limerio) [inline apps/admin/src/App.tsx]:
```suggestion
		<></>
```

File: apps/admin/src/main.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from "react"
+import { createRoot } from "react-dom/client"
+import App from "./App.tsx"
+import "./index.css"
```
Comment (Limerio) [inline apps/admin/src/main.tsx]: remove this line

File: apps/admin/tsconfig.app.json
Comment (Limerio) [inline apps/admin/tsconfig.app.json:1]: remove the comments

File: apps/admin/tsconfig.node.json
Comment (Limerio) [inline apps/admin/tsconfig.node.json:1]: remove the comments

File: apps/api/README.md
Comment (Limerio) [inline apps/api/README.md:1]: Empty the file and remplace the content by this
```md
# API
```

File: apps/admin/vite.config.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vite"
+import react from "@vitejs/plugin-react"
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [react()],
```
Comment (Limerio) [inline apps/admin/vite.config.ts]: use the [vite-tsconfig-paths](https://www.npmjs.com/package/vite-tsconfig-paths) plugin

File: apps/api/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../tsconfig.json"],
+
```
Comment (Limerio) [inline apps/api/tsconfig.json]:
```suggestion
```

File: apps/landing/README.md
Comment (Limerio) [inline apps/landing/README.md:1]: Empty the file en remplace by this
```md
# Landing
```

File: apps/landing/index.html
Comment (Limerio) [inline apps/landing/index.html:1]: Change the content title tag to `Landing - ALittleByte`

File: apps/landing/public/vite.svg
Comment (Limerio) [inline apps/landing/public/vite.svg:1]: Remove this file

File: apps/landing/src/App.css
Comment (Limerio) [inline apps/landing/src/App.css:1]: Remove this file

File: apps/landing/src/assets/react.svg
Comment (Limerio) [inline apps/landing/src/assets/react.svg:1]: Remove this file

File: apps/landing/src/index.css
Comment (Limerio) [inline apps/landing/src/index.css:1]: Remove this file

File: apps/landing/src/main.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from "react"
+import { createRoot } from "react-dom/client"
+import App from "./App.tsx"
+import "./index.css"
```
Comment (Limerio) [inline apps/landing/src/main.tsx]: Remove this line

File: apps/landing/src/App.tsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
+import { useState } from "react"
+import reactLogo from "./assets/react.svg"
+import viteLogo from "/vite.svg"
+import "./App.css"
```
Comment (Limerio) [inline apps/landing/src/App.tsx]: remove those lines

File: apps/landing/src/App.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+function App() {
+	const [count, setCount] = useState(0)
```
Comment (Limerio) [inline apps/landing/src/App.tsx]: remove this line

File: apps/landing/src/App.tsx [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+				</button>
+				<p>
+					Edit <code>src/App.tsx</code> and save to test HMR
+				</p>
+			</div>
+			<p className="read-the-docs">
+				Click on the Vite and React logos to learn more
+			</p>
+		</>
+	)
```
Comment (Limerio) [inline apps/landing/src/App.tsx]:
```suggestion
	return <></>
```

File: apps/landing/tsconfig.app.json
Comment (Limerio) [inline apps/landing/tsconfig.app.json:1]: Remove the comments

File: apps/landing/tsconfig.json [hunk 11]
```diff
@@ -0,0 +1,12 @@
+{
+  
+  "exclude": ["node_modules", "dist"],
+  "include": ["**/*.tsx", "**/*.ts", "src/vite-env.d.ts"],
+
```
Comment (Limerio) [inline apps/landing/tsconfig.json]:
```suggestion
```

File: apps/landing/tsconfig.node.json
Comment (Limerio) [inline apps/landing/tsconfig.node.json:1]: Remove the comments

File: apps/landing/vite.config.ts [hunk 12]
```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vite"
+import react from "@vitejs/plugin-react"
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [react()],
```
Comment (Limerio) [inline apps/landing/vite.config.ts]: use the [vite-tsconfig-paths](https://www.npmjs.com/package/vite-tsconfig-paths) plugin

**Review by Limerio — CHANGES_REQUESTED** [review]
Don't forget to format the code

**Review by Limerio — APPROVED** [review]

**Pull request merged**